### PR TITLE
Implementing UI

### PR DESCRIPTION
**Pull request by alexandragnj**
Screenshot_1659083908
Screenshot_1659084292
Screenshot_1659084308
Screenshot_1659084312

### CODE REVIEW

File: app/src/main/java/com/example/criminalintentapp/presentation/fragments/authentication/LoginFragment.kt [hunk 1]
```diff
@@ -28,15 +46,34 @@ class LoginFragment : Fragment() {
...
+        (requireActivity() as AppCompatActivity).supportActionBar?.hide()
+
+        progressDialog = ProgressDialog(requireActivity())
 
         if (authenticationViewModel.currentUser != null) {
             goToCrimeList()
         }
 
+        googleLogin()
+
```
Comment (iugaliviu) [inline app/src/main/java/com/example/criminalintentapp/presentation/fragments/authentication/LoginFragment.kt:58]: Please remove this empty line

**Review by iugaliviu — COMMENTED** [review]
Some small details that need to be addressed. It's the details that make the difference. It may not look like it, but proper formatting, naming and placement of the code can make the okish code into good code, and then great code

File: app/src/main/java/com/example/criminalintentapp/presentation/fragments/authentication/LoginFragment.kt [hunk 2]
```diff
@@ -74,7 +121,13 @@ class LoginFragment : Fragment() {
...
 
     private fun goToCrimeList() {
+        progressDialog.hideProgressDialog()
         NavHostFragment.findNavController(this)
             .navigate(R.id.action_loginFragment_to_crimeListFragment)
     }
+
+    companion object {
+        const val TAG = "LoginFragment"
+        private const val RC_SIGN_IN = 120
```
[the remark this answers is not in the document]
Comment (alexandragnj) [reply app/src/main/java/com/example/criminalintentapp/presentation/fragments/authentication/LoginFragment.kt:131]: request code

[the remark this answers is not in the document]
Comment (iugaliviu) [reply app/src/main/java/com/example/criminalintentapp/presentation/fragments/authentication/LoginFragment.kt:139]: Please rename it so it reflects its purpose: (E.g: REQUEST_CODE_GOOGLE_SIGN_IN)

**Review by iugaliviu — APPROVED** [review]
Beautiful

**Pull request merged**